src/smcache/lookup.rs

```diff
@@ -124,30 +174,6 @@ impl<'data> SmCache<'data> {
 
         Some(SourceLocation { file, line, scope })
     }
-
-    /// Returns the [`File`] which allows fast access to source lines.
-    pub fn get_file(&self, name: &str) -> Option<File> {
```

Comment:
I would actually keep this getter. You might want to still get access to the files/sourcesContent from the sourcemap.

Alternatively, we can also expose an iterator over all the files. Then the API user can collect that into a Map themselves. I think that might be the best solution even.